modules/conflict_resolver/help/conflict_resolver.md

```diff
@@ -0,0 +1,9 @@
+# Conflict Resolver
+
+This module is designed to identify and resolve data conflicts in a study. LORIS uses double data entry, which means you may be asked to enter critical data twice in duplicate instrument forms. The Conflict Resolver will flag any discrepancies between the two entries (called *unresolved conflicts*) and allow you to fix—resolve—them easily. 
```

Comment:
```suggestion
This module is designed to identify and resolve data conflicts in a study. LORIS uses double data entry, which means you may be asked to enter critical data twice in duplicate instrument forms. The Conflict Resolver will flag any discrepancies between the two entries (called *unresolved conflicts*) and allow you to fix or "resolve" them easily. 
```

GitHub is showing the dash as a hyphen in the diff. I'm not sure how the font in LORIS displays it, but it's probably better to just use words and avoid the possibility of user's wondering what a fix-resolve is.

Reply:
hm, I tried committing your suggestion but am getting the message "You do not have permission to push to this repository"

Reply:
@johnsaigle based on the PR I think "this repository" is referring to your fork in the error message